### DISCUSSION TITLE
astro command to generate a markdown file

### DISCUSSION THREAD

**Discussion by kresnasatya** [withastro/docs#2979] [Ideas]
Hi, is that astro has a command to generate a markdown file e.g. for blog post) with command line like hugo.io? If you’re using hugo it looks like: hugo new content posts/awesome-post.md. the posts refer a directory where hugo will generate a file. I wish astro have this command.

**Comment by yuxxeun** [comment]
i have `shell` script to do this job, it work fine whenever im using with next.js.
sadly, for astro, i cannot running that script well, i dont know why, maybe we can solve it together?

**Comment by delucis** [comment]
There’s some discussion about this in the Astro roadmap here that you might find interesting: https://github.com/withastro/roadmap/discussions/553

I’ll close the discussion here as this is the repo for the Astro documentation site and the roadmap repo is where we focus our brainstorming and feature proposals.

**Reply by yuxxeun** [reply]
thanks, i'll move this topic to there
